presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -144,6 +144,11 @@ public boolean isLeaf()
         return remoteSourceNodes;
     }
 
+    public boolean hasDefaultOutput()
```

Comment:
If this is about scheduling empty splits, let's call it something related to that instead of beating around the bush